app/src/main/res/layout/item_channel.xml

```diff
@@ -10,7 +10,8 @@
 
```

Comment:
this line causing `android:layout_margin="@dimen/space_16x"` top and bottom margins too big can you change it to 
```
android:layout_marginStart="@dimen/space_16x"
android:layout_marginTop="@dimen/space_4x"
android:layout_marginEnd="@dimen/space_16x"
android:layout_marginBottom="@dimen/space_16x"
```